symfit/core/fit.py

```diff
@@ -1390,8 +1390,8 @@ def _determine_objective(model, objective, minimizer, bound_arguments):
                 # be considered separately and has its own non standard
                 # objective function.
                 return VectorLeastSquares
-            if len(model) == 1:
-                if len(model.independent_vars) == 0 and model.dependent_vars[0].name not in bound_arguments.arguments:
+            if len(model) == 1 and len(model.independent_vars) == 0:
+                if model.dependent_vars[0].name not in bound_arguments.arguments:
```

Comment:
I still think it's better to combine them:
```
if (len(model) == 1 and  not model.independent_vars
        and model.dependent_vars[0].name not in bound_arguments.argument):
```
Or make a variable out of some of them.

Reply:
You're right, that is better. After reading PEP 8 on this it seems like that is the solution. PyCharm automatically breaks long lines with `\`, which I really dislike and try to avoid at all cost. But I forgot the parentheses solve this problem in a very clean way.